### PR TITLE
Updating llama2 building steps in README.md

### PR DESCRIPTION
**Pull request by rutu-k**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,60 @@ Generate Kubernetes resource YAML manifests from a text prompt
...
+1. Request for accessing Llama models [here](https://ai.meta.com/resources/models-and-libraries/llama-downloads/)
+![image](https://github.com/PrasadG193/Gener8-Llama2/assets/25836028/17f92fa9-db87-41f0-ac82-7c9ad818f2fb)
+
+You will receive a mail with the URL to download the model which we will use later.
+![image](https://github.com/PrasadG193/Gener8-Llama2/assets/25836028/86e123ea-36d8-4604-a61f-5f163d665f15)
 
 
-### Converting Downloaded Model
+3. Request for accessing Llama models on [HuggingFace](https://huggingface.co/meta-llama/Llama-2-7b-hf) and clone the repository
+![image](https://github.com/PrasadG193/Gener8-Llama2/assets/25836028/250f597a-8038-4b06-b652-5cf939f472b2)
```
Comment (PrasadG193) [inline README.md]: This is not required

File: README.md [hunk 2]
```diff
@@ -3,15 +3,60 @@ Generate Kubernetes resource YAML manifests from a text prompt
...
+   
+### Converting and Quantizing Downloaded Model
+Now we have to convert the downloaded model to f16 format and quantize it to reduce its size.
+1. First activate a virtual env and install all the requirements
+   ```sh
+   $ cd ../llama.cpp
+   $ python3 -m venv llama2
+   $ source llama2/bin/activate
+   $ python3 -m pip install -r requirements.txt
+   ```
```
Comment (PrasadG193) [inline README.md]:
```suggestion
1. Build llama.cpp project
    ```
     $ cd llama.cpp
     $ make
    ```
2. First activate a virtual env and install all the requirements
   ```sh
   $ cd ../llama.cpp
   $ python3 -m venv llama2
   $ source llama2/bin/activate
   $ python3 -m pip install -r requirements.txt
   ```
```

File: README.md (hunk 1 above)
Comment (rutu-k) [reply README.md]: resolved

File: README.md [hunk 3]
```diff
@@ -3,15 +3,58 @@ Generate Kubernetes resource YAML manifests from a text prompt
...
+
+3. Then convert the model into f16 format and quantize it
+   ```sh
+   $ python3 convert.py --outfile models/7B-chat/ggml-model-f16.bin --outtype f16 ../../llama2/llama/llama-2-7b-chat --vocab-dir ../../llama2/llama
+   $ ./quantize  ./models/7B-chat/ggml-model-f16.bin ./models/7B-chat/ggml-model-q4_0.bin q4_0
+   ```
+3. Make sure you change the `vocab_size` in llama/llama-2-7b-chat/params.json to 32000
```
Comment (PrasadG193) [inline README.md]:
```suggestion
4. Make sure you change the `vocab_size` in llama/llama-2-7b-chat/params.json to 32000
```

File: README.md [hunk 4]
```diff
@@ -3,15 +3,58 @@ Generate Kubernetes resource YAML manifests from a text prompt
...
+![image](https://github.com/rutu-k/Gener8-Llama2/assets/25836028/410311c5-599b-4bdd-b1e3-cf3f7aacdc15)
+
+
 ## Prerequisites
 
-### Setup Llama2 Model
+Please make you have Python 3.8.X or higher version
 
 ### Requesting access to Llama Models
+1. Request for accessing Llama models [here](https://ai.meta.com/resources/models-and-libraries/llama-downloads/)
```
Comment (PrasadG193) [inline README.md]:
```suggestion
Request for accessing Llama models [here](https://ai.meta.com/resources/models-and-libraries/llama-downloads/)
```

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**